src/lib/storage/storage_manager.cpp

```diff
@@ -10,38 +10,59 @@
 namespace opossum {
 
 StorageManager& StorageManager::get() {
```

Comment:
In theory, we could add a test for this method (that it actually returns the same instance every time). With the old code, this test would then fail. Do we want this?

Reply:
I think it is unecessary, as even when it does this 2 times (what we could test), this would not give too much information.

Reply:
I change my opinion: The test would already fail with the old code, since if the `get` method returned a new instance every time, tests for e.g. `get_table` could not work.